models/classes/class.QtiTestService.php

```diff
@@ -570,6 +570,13 @@ protected function importTest(core_kernel_classes_Class $targetClass, Resource $
                 // If any, assessmentSectionRefs will be resolved and included as part of the main test definition.
                 $testDefinition->includeAssessmentSectionRefs(true);
 
+
+                if ($overwriteTest) {
```

Comment:
Something that we might be missing here is that `assets` are imported as base64, so when authoring the items after reimporting the assets (media manager) reference is not existing anymore. 

I hope this will not be an issue, but just a heads up in case we have some use case for this. 

cc: @ar0oat @yaraslau-kavaliou

Reply:
thanks, we need to keep an eye on this.

Reply:
Maybe we can clarify it with @julien-sebire or some BA. I cannot say for sure if this will be an issue for the authors, but if it is, we would have to change how we export/import tests/item/assets in tao.

Reply:
So after we export from authoring.

- We lost any relation with shared stimulus/media ID
- If a shared stimulus has a media inside, it becomes base64


Conclusion, by **importing the test back** it will not keep relations with media, so any media / shared stimulus becomes part of this very single item.